### PR TITLE
Update readme for improved readability

### PR DESCRIPTION
**Pull request by bbowman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ To use the bridge you'll need:
 - **Windows 10**, build 10586 or higher. Validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system).
 - **Visual Studio 2017** with Windows developer tools. Visual Studio 2017 Community is available for free [here](https://dev.windows.com/downloads).
 
-   Ensure the following individual components are selected during installation (Selecting the **Universal Windows Platform development** workflow should give most of the below):
+  
+   <details><summary>Ensure the following individual components are selected during installation. Selecting the Universal Windows Platform development workflow should give most of the below (Click to Expand)</summary><p>
```
Comment (jaredhms) [inline README.md]: >Click to Expand [](start = 191, length = 15)

nit: might make sense to make this look like a link; the 'click to expand' didn't really jump out at me at first

**Comment by jaredhms** [comment]
:shipit:

File: README.md (hunk 1 above)
Comment (bbowman) [inline README.md]: Not sure what you mean by "make it look like a link". I can't do markdown in the summary field it doesn't look like.

**Comment by andrejo-msft** [comment]
> choco upgrade winobjc-tools

Needs to be 'install winobjc-tools -pre'.

---
Refers to: README.md:55 in 27b298b. [](commit_id = 27b298bef78874e8b680ac481490c7a2d17b812f, deletion_comment = False)

**Comment by bbowman** [comment]
> choco upgrade winobjc-tools

That is only if we want to direct people towards our pre-release packages which I don't think we should.

---
In reply to: [288177498](https://github.com/Microsoft/WinObjC/pull/2283#issuecomment-288177498) [](ancestors = 288177498)

---
Refers to: README.md:55 in 27b298b. [](commit_id = 27b298bef78874e8b680ac481490c7a2d17b812f, deletion_comment = False)

**Comment by andrejo-msft** [comment]
>   - Static analysis tools

I appreciate this list, but the discovery on this is difficult through the light-weight installer. Could we simply state 'Select Universal Windows Platform development workflow' and then add the individual components that remain?

---
Refers to: README.md:36 in 27b298b. [](commit_id = 27b298bef78874e8b680ac481490c7a2d17b812f, deletion_comment = False)

File: README.md [hunk 2]
```diff
@@ -127,6 +116,18 @@ See our [contribution guidance in the wiki](https://github.com/Microsoft/WinObjC
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+## Advanced Installation for Contributors and Ninjas
```
Comment (andrejo-msft) [inline README.md]: 'Ninjas'? :)

**Comment by bbowman** [comment]
>   - Static analysis tools

I'm confused. Thats exactly what I'm saying? Or are you saying that you want the list to *only* include the items not covered by that workflow?

---
In reply to: [288178049](https://github.com/Microsoft/WinObjC/pull/2283#issuecomment-288178049) [](ancestors = 288178049)

---
Refers to: README.md:36 in 27b298b. [](commit_id = 27b298bef78874e8b680ac481490c7a2d17b812f, deletion_comment = False)

File: README.md (hunk 2 above)
Comment (bbowman) [inline README.md]: I had fun.

---
In reply to: [107240965](https://github.com/Microsoft/WinObjC/pull/2283#discussion_r107240965) [](ancestors = 107240965)

File: README.md (hunk 1 above)
Comment (jaredhms) [inline README.md]: bold text would be a good start

**Comment by asimonov-msft** [comment]
>   - Visual C++ copmilers and libraries for ARM

typo

---
Refers to: README.md:40 in 27b298b. [](commit_id = 27b298bef78874e8b680ac481490c7a2d17b812f, deletion_comment = False)

**Comment by asimonov-msft** [comment]
:shipit:

**Comment by andrejo-msft** [comment]
>   - Static analysis tools

The list should contain the core component and the additions needed to give the developer the analysis tools that they need.

In general, we should assume that this is the developer's first time with Visual Studio.

---
In reply to: [288179569](https://github.com/Microsoft/WinObjC/pull/2283#issuecomment-288179569) [](ancestors = 288179569,288178049)

---
Refers to: README.md:36 in 27b298b. [](commit_id = 27b298bef78874e8b680ac481490c7a2d17b812f, deletion_comment = False)

**Pull request merged**